app/src/main/java/ca/bc/gov/bchealth/ui/auth/BcServicesCardLoginFragment.kt

```diff
@@ -0,0 +1,149 @@
+package ca.bc.gov.bchealth.ui.auth
+
+import android.os.Bundle
+import android.view.View
+import androidx.activity.OnBackPressedCallback
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material.Icon
+import androidx.compose.material.IconButton
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Scaffold
+import androidx.compose.material.contentColorFor
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.fragment.app.activityViewModels
+import androidx.navigation.fragment.findNavController
+import ca.bc.gov.bchealth.R
+import ca.bc.gov.bchealth.compose.MyHealthTheme
+import ca.bc.gov.bchealth.model.BcServiceCardLoginInfoType
+import ca.bc.gov.bchealth.ui.BaseFragment
+import ca.bc.gov.bchealth.ui.custom.MyHealthToolBar
+import ca.bc.gov.bchealth.ui.healthrecord.HealthRecordPlaceholderFragment
+import ca.bc.gov.bchealth.ui.healthrecord.NavigationAction
+import ca.bc.gov.bchealth.ui.login.BcscAuthFragment
+import ca.bc.gov.bchealth.ui.login.BcscAuthState
+import ca.bc.gov.bchealth.viewmodel.SharedViewModel
+import dagger.hilt.android.AndroidEntryPoint
+
+/**
+ * @author Pinakin Kansara
+ * [BcServicesCardLoginFragment] is representing state when the user is not logged in.
+ * This screen is designed with compose & in future this might require some modification
+ * that makes it a reusable screen to handle logout state.
+ *
+ * Following places in the code require to be updated in future
+ * - [ca.bc.gov.bchealth.ui.healthrecord.individual.IndividualHealthRecordFragment]
+ * - [ca.bc.gov.bchealth.ui.dependents.DependentsFragment]
+ */
+@AndroidEntryPoint
+class BcServicesCardLoginFragment : BaseFragment(null) {
```

Comment:
Sonar is failing because this and the previous Fragment have code duplication:
https://sonarcloud.io/component_measures?id=bcgov_myhealthBC-android&pullRequest=467&metric=new_duplicated_lines_density&view=list

What do you think of create a  BcServicesCardFragment with the common code and reuse it?

Reply:
This will be a major refactor and with compose adoption it will go away. this is temporary state